Volume/Models/Publication.swift

```diff
@@ -9,10 +9,12 @@
 import Foundation
 
 struct Publication: Hashable, Identifiable {
+    var id: String {
+        slug
+    }
```

Comment:
The `id` property is required by the `Identifiable` protocol and allows you to distinguish different instances of `Publication`. I made it a computed variable that always returns the `slug` because it's assigned to be unique anyway. And naturally, computed variables have to be marked as `var` and not `let`.